xdp-dump/tests/test-xdpdump.sh

```diff
@@ -228,6 +231,8 @@ test_capt_term()
 
 test_exitentry()
 {
+    assert_kernel_version_greater_or_equal 5.7.0
```

Comment:
You should be able to make this test pass with the older kernel, guess only lines 236-240 will probably need to be conditional.